README.md

```diff
@@ -1,26 +1,21 @@
 # BuyTogether
+Welcome to our group social project!
+In this project, we have developed a web application that is supposed to help people group up,
+and organize a joint food purchase.
 
-Welcome to our group social project!.
-In this project we have developed a web application which suppose to help people group up
-and organize a large scale food buy.
-Doing so, we "cut" the amount of hands the product go through which will benefit the customer
-with cheaper prices.
-
+Doing so, we remove the middlemen and reduce the price of the food products.
 ---
+## Team members
 
-# Team members
 * Paz Menachem
 * Lior Silberman
 * Meitar Rizner
 * Rawan Abuleil
 * Karam Jbareen
-
 ---
-
-# Contribute
-In order to contribute please refer to the contributing guide below.
-
+## Contribute
+In order to contribute, please refer to the contributing guide below.
+* [Contribute] (https://github.com/redhat-beyond/beyond08-group3/blob/main/CONTRIBUTING.md/)
```

Comment:
The link doesn't get renamed. I think the problem is the extra space. Also, you can use relative path: https://github.blog/2013-01-31-relative-links-in-markup-files/